kfs-core/src/main/resources/institutional-config.properties

```diff
@@ -59,11 +59,11 @@ spring.source.files=classpath:spring-kfs-imported-rice-beans.xml, \
   classpath:edu/arizona/kfs/fp/spring-fp.xml, \
```

Comment:
For some reason, leaving classpath:edu/arizona/kfs/module/cg/spring-cg.xml at line 62 would make the spring file not able to read the cgModuleConfiguration bean, it won't throw an error but the new Award.xml and AwardExtension.xml were not being read; even when I purposely misspelled the parent="cgModuleConfigurateion-parentBean", the spring-cg would not throw an error. 
However, when moving this line at the bottom of the list then everything will work as expected. It would throw an exception when purposely misspelling the parent bean on the spring file.